DQMOffline/Trigger/interface/HLTTauDQMOfflineSource.h

```diff
@@ -55,6 +58,10 @@ class HLTTauDQMOfflineSource : public DQMEDAnalyzer {
       edm::EDGetTokenT<LVColl> token;
     };
     std::vector<RefObject> refObjects_;
+    std::vector<edm::ParameterSet> tagAndProbePaths;
```

Comment:
in general it is preferable to use `not collection.empty()` instead of `collection.size() != 0`